pkg/util/cloudproviders/cloudfoundry/types.go

```diff
@@ -276,6 +301,10 @@ func DesiredLRPFromBBSModel(bbsLRP *models.DesiredLRP, includeList, excludeList
 			} else {
 				log.Debugf("Could not find org %s in cc cache", orgGUID)
 			}
+			if sidecars, err := ccCache.GetSidecars(appGUID); err == nil && len(sidecars) > 0 {
```

Comment:
Should we put this behind a feature flag?

Reply:
Yes we should, it makes sense to keep it optional (reduce api calls) and expose it under "Enable Cloud Foundry API apps polling" in the tile.

Reply:
In this case, I'll tag start adding `sidecar_present:false` as well.